src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -64,7 +69,34 @@ void setMessageStore(MessageStore messageStore) {
     this.messageStore = messageStore;
   }
 
-  /**
+  public static class DateFormatter {
+
+    static final String FORMAT = "EEE MMM dd yyyy hh:mm:ss a z";
+
+    /**
+     * Utility function to convert java Date to TimeZone format
+     *
+     * @param date date to be formatted
+     */
+    public static String formatDateToString(Date date) {
+      String timeZone = "PST";
```

Comment:
make this into a static final constant too

Reply:
Done